docker-compose.yml

```
@@ -23,12 +23,64 @@ services:
+      MYSQL_DATABASE: OCM_task
+    ports:
+      - "3309:3306"
+    networks:
+      - mynetwork
+
+  mysql-submission:
+    image: mysql:8
+    container_name: mysql-submission
+    restart: always
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: yes
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)